client/app/dedicated/server/interfaces/ola-configuration/ola-configuration.html

```diff
@@ -0,0 +1,188 @@
+        </a>
+    </p>
+</oui-message>
+
+<oui-message class="mb-3" data-type="warning">
+    <p class="mb-0" data-translate="dedicated_server_interfaces_ola_alert_configuration_reset"></p>
+</oui-message>
+
+<oui-stepper
+    data-on-finish="$ctrl.onFinish()">
+    <oui-step-form
+        data-header="{{:: 'dedicated_server_interfaces_ola_step1_title' | translate }}"
+        data-description="{{:: 'dedicated_server_interfaces_ola_field_mode_description' | translate }}">
+        <oui-field data-label="{{:: 'dedicated_server_interfaces_ola_field_mode_label' | translate }}">
+            <div class="row">
+                <div class="col-md-4"
+                    data-ng-repeat="mode in $ctrl.olaModes track by $index">
+                    <oui-radio
+                        class="h-100"
+                        name="mode"
+                        data-description="{{:: 'dedicated_server_interfaces_ola_field_mode_choice_' + mode + '_description' | translate}}"
+                        data-disabled="$ctrl.isModeDisabled(mode)"
+                        data-model="$ctrl.configuration.mode"
+                        data-value="mode"
+                        data-thumbnail
+                        data-required>
+                        <span data-translate="{{ 'dedicated_server_interfaces_ola_field_mode_choice_' + mode }}"></span>
+                    </oui-radio>
+                </div>
+            </div>
+        </oui-field>
+        <oui-field data-label="{{:: 'dedicated_server_interfaces_ola_field_name_label' | translate}}"
+            data-size="l">
+            <input class="oui-input"
+                name="name"
+                type="text"
+                autocomplete="off"
+                placeholder="{{:: 'dedicated_server_interfaces_ola_field_name_placeholder' | translate }}"
+                data-ng-model="$ctrl.configuration.name"
+                data-required>
+        </oui-field>
+    </oui-step-form>
+    <oui-step-form
+        data-header="{{:: 'dedicated_server_interfaces_ola_step2_title' | translate }}"
+        data-description="{{:: 'dedicated_server_interfaces_ola_step2_description' | translate }}"
+        data-valid="!!$ctrl.selectedInterfaces.length && $ctrl.selectedInterfaces.length > 1"
+        data-prevent-next>
+
+        <!-- Obsolete bandwidth option -->
+        <oui-message class="mb-3"
+            data-ng-if="$ctrl.hasObsoleteBandwithOption()"
+            data-type="warning">
+            <p class="mb-0" data-translate="dedicated_server_interfaces_ola_alert_obsolete_option"></p>
+        </oui-message>
+
+        <!-- Forbidden interface -->
+        <oui-message class="mb-3"
+            data-ng-if="$ctrl.notAllowedInterfaces.length"
+            data-type="warning">
+            <p data-translate="dedicated_server_interfaces_ola_alert_not_allowed"></p>
+            <ul class="mb-0">
+                <li
+                    data-ng-repeat="interface in $ctrl.notAllowedInterfaces"
```

Comment:
no `track by` ?

Reply:
We could here, it's not something that will be changed during the configuration